src/script/components/MessagesList/MessageList.tsx

```diff
@@ -277,13 +277,16 @@ export const MessagesList: FC<MessagesListParams> = ({
           }
           const {messages, firstMessageTimestamp} = group;
 
-          return messages.map((message, index) => {
+          return messages.map(message => {
             const isLastDeliveredMessage = lastDeliveredMessage?.id === message.id;
 
             const visibleCallback = getVisibleCallback(conversation, message);
 
             const key = `${message.id || 'message'}-${message.timestamp()}`;
 
+            const isHighlighted = !!highlightedMessage && highlightedMessage === message.id;
+            const isFocused = !!focusedId && focusedId === message.id;
```

Comment:
This is the actual bugfix. We need to check `!!focusedId` before doing the comparison between `focusedId` and `message.id`. 

In case `focusedId === undefined` then this condition would be true